OneCommunityGlobal/HighestGoodNetworkApp
# Sheetal M - Add unit test for teamsTeamMembersReducer File

# Description
Adding unit testcases for teamsTeamMembersReducer.js 

## Related PRS (if any):
No Related PRs

## Main changes explained:
- Added a new file under reducers/tests called teamsTeamMembersReducer.test.js
- The list of test cases for teamsTeamMembersReducer
  - return the initial value when no action is specified
  - handles RECEIVE_TEAM_USERS action
  - handle TEAM_MEMBER_ADD action
  - handle TEAM_MEMBER_DELETE action
  - handle FETCH_TEAM_USERS_START action
 
## How to test:
1. Checkout current batch
2. Run npm test teamsTeamMembersReducer.test.js
3. Verify all test cases passing without any errors 

## Screenshots or videos of changes:

![teamsTeamMembersReducer_test_proof](https://github.com/user-attachments/assets/78796cc9-bffd-4992-a426-fd663e460481)
